CvGameCoreDLL/CvCity.cpp

```diff
@@ -2867,14 +2867,22 @@ int CvCity::getProductionExperience(UnitTypes eUnit)
 
 			//Leoreth: Harmandir Sahib effect
             if (GET_PLAYER(getOwnerINLINE()).isHasBuilding((BuildingTypes)HARMANDIR_SAHIB))
+			{

```

Comment:
The indentation here looks wrong.

Reply:
Well spaces and tabs are different, we should be consistent. Looks like tabs is the default in this file.